exercises/practice/high-scores/.meta/example.rb

```diff
@@ -18,8 +18,4 @@ def latest
   def personal_top_three
     scores.max(3)
   end
-
-  def latest_is_personal_best?
-    latest == personal_best
-  end
```

Comment:
That is unfortunate, as it provides a space for me to talk about attributes macros in Ruby, and how to maintain the ability to have a method generated in C, while still having a question mark at the end of the name of the method…

Not the goal of the exercise, of course, but a "So you want to dig deeper into how the language works, because the problem is a simple one?" kind of side stretch.